src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
```

Comment:
The variable name 'clone' could be more descriptive. Consider renaming it to 'invertedRobot' to better reflect its purpose as the inverse of the input 'robot'.